docs/api/blockchain/state_channels.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: state_channels
+sidebar_label: State Channels
+slug: /api/blockchain/state_channels
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# State Channels
+
+---
+
+## State Channel Closes
+
+```
+GET https://api.helium.io/v1/state_channels
+```
+
+Use this API to list state channel close transactions.
+
+This route is `cursor` paginated.
+
+<Tabs
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
+_Query Parameters_
+
+| param             | Type     | Note                     |
+| ----------------- | -------- | ------------------------ |
+| cursor (optional) | _string_ | Cursor for page to fetch |
+
+</TabItem>
+<TabItem value="response">
+
+200: OK
+
+Return a page of state channel close transactions. Only a fraction of the
```

Comment:
might want to rephrase or remove the part about "fraction of the summaries" shown because I thought that the endpoint only returned a fraction of the summaries